probe_scraper/probe_expiry_alert.py

```diff
@@ -120,20 +129,37 @@ def parse_args():
 
 
 def main(current_date, dryrun):
-    probe_info = requests.get(PROBE_INFO_BASE_URL + "firefox/all/main/all_probes").json()
-
     current_version = get_latest_nightly_version()
     next_version = str(int(current_version) + 1)
 
-    expired_probes = find_expiring_probes(probe_info, current_version)
-    expiring_probes = find_expiring_probes(probe_info, next_version)
+    events_file_path = os.path.join(tempfile.gettempdir(), "Events.yaml")
+    download_file(BASE_URI + EVENTS_FILE, events_file_path)
+    events = EventsParser().parse([os.path.join(tempfile.gettempdir(), events_file_path)])
+
+    histograms_file_path = os.path.join(tempfile.gettempdir(), "Histograms.json")
+    download_file(BASE_URI + HISTOGRAMS_FILE, histograms_file_path)
+    histograms = HistogramsParser().parse(
+        [os.path.join(tempfile.gettempdir(), histograms_file_path)],
+        version=next_version
+    )
+
+    scalars_file_path = os.path.join(tempfile.gettempdir(), "Scalars.yaml")
+    download_file(BASE_URI + SCALARS_FILE, scalars_file_path)
+    scalars = ScalarsParser().parse([os.path.join(tempfile.gettempdir(), scalars_file_path)])
```

Comment:
this should reuse `scalars_file_path`
```suggestion
    scalars = ScalarsParser().parse([scalars_file_path])
```